cfgov/regulations3k/jinja2/regulations3k/regulations3k-search-bar.html

```diff
@@ -22,7 +22,7 @@
             </div>
         </div>
         <div class="o-form__input-w-btn_btn-container">
-            <button class="a-btn">Search</button>
+            <button class="a-btn" aria-label="Search for term in regulations">Search</button>
```

Comment:
@sonnakim I added this more descriptive label for http://localhost:8000/rules-policy/regulations/search-regulations/results/?regs=1041 Let me know if this should be different.

Reply:
Thanks! I feel like "Search regulations" would be pretty understandable on its own? If not, maybe we could make the sentence "Search for a term in the regulations"?